### PR TITLE
Packet.net nixops deployments

### PR DESCRIPTION
**Pull request by disassembler**
Removes legacy packet.net machines and replaces them with packet.net nixops module.

### CODE REVIEW

File: deployments/packet-net.nix [hunk 1]
```diff
@@ -2,14 +2,43 @@
...
+
     imports = [ ../modules/common.nix
                 module
-              ] ++
-              # This loads the packet.net-provided .nix files.
-              map (f: builtins.toPath(toString(./.) + "/${hostname}/" + f))
-                  (builtins.attrNames (builtins.readDir (./. + "/${hostname}")));
+              ] ++ (optionals (type == "legacy")
+              (map (f: builtins.toPath(toString(./.) + "/${hostname}/" + f))
+                  (builtins.attrNames (builtins.readDir (./. + "/${hostname}")))));
```
Comment (cleverca22) [inline deployments/packet-net.nix]: you probably dont want to use `toString` or `toPath`
just `./. + "/${hostname}/${f}` should do

Comment (johnalotoski) [reply deployments/packet-net.nix]: Updated

**Pull request merged**